src/main/scala/Top.scala

```diff
@@ -3,7 +3,9 @@ import chisel3._
 class Top extends Module {
   val io = IO(new Bundle {
     val gpio = Output(UInt(32.W))
+    val reset = Input(Bool())
```

Comment:
IMO this is an FPGA specific behaviour, we should not include this in source tree.